docs/src/starknet/index.md

```diff
@@ -51,14 +50,16 @@ response: [0x0]
 > In the above example we supply `sncast` with `--account` and `--url` flags. If `snfoundry.toml` is present, and have these properties set, values provided using these flags will override values from `snfoundry.toml`. Learn more about `snfoundry.toml` configuration [here](../projects/configuration.md#sncast).
 
 
-### Calldata
+### Arguments and Calldata
 
 Some `sncast` commands (namely `call`, `deploy` and `invoke`) allow passing *calldata* - a series of arguments to perform an action with on blockchain.
 
-In the example above we called a function with an argument: `0x0`, passed using `--calldata` flag.
+In the example above we called a function with an argument: `'"Charizard"'`, passed using `--arguments` flag, but it can
+be also obtained by passing a serialized form, in this example equivalent to passing `--calldata` with the following value: `0x0 0x43686172697a617264 0x9`.
 
 Please note the notation of the argument. The default way of passing calldata is a list of hexadecimally encoded field elements - the *serialized* calldata.
 To obtain the serialized form of the wished data, one must write a Cairo program calling `Serde::serialize` on subsequent arguments and displaying the results.
```

Comment:
This is outdated, serialized is no longer default - there is no default in code at all.